easybuild/easyblocks/c/cmake.py

```diff
@@ -95,6 +130,7 @@ def configure_step(self):
             self.log.info("Found sysroot '%s', adding it to $CMAKE_PREFIX_PATH and $CMAKE_LIBRARY_PATH", sysroot)
             cmake_prefix_path.append(sysroot)
             cmake_library_path.append(os.path.join(sysroot, 'usr', 'lib'))
+            cmake_library_path.append(os.path.join(sysroot, 'usr', 'lib64'))
```

Comment:
I wonder if this chunk is still needed at all: if CMake is patched itself do you still need those environment variables set?

Reply:
So I believe this chunk should be removed since these directories are already searched via the compiled-in CMAKE_SYSTEM_PREFIX_PATH.

Reply:
They should actually never be required. CMake already reroots paths when the SYSPATH or FIND_ROOT_PATH is passed